src/styles/getModalStyles.js

```diff
@@ -69,7 +69,31 @@ export default (type, windowDimensions) => {
             swipeDirection = undefined;
             animationIn = 'slideInUp';
             animationOut = 'slideOutDown';
-            needsSafeAreaPadding = false;
+            break;
+        case CONST.MODAL.MODAL_TYPE.POPOVER:
+            modalStyle = {
+                margin: 0,
+                alignItems: 'center',
+                justifyContent: 'flex-end',
+                marginRight: windowDimensions.width - variables.sideBarWidth,
+                marginBottom: 82,
+            };
+            modalContainerStyle = {
+                width: variables.sideBarWidth - 40,
+                borderRadius: 12,
+                borderWidth: 1,
+                borderColor: '#ECECEC',
```

Comment:
Can you use the color constant here? https://github.com/Expensify/Expensify.cash/blob/d93fe7ca7585bdcdfcbdabb2d6fd2a639d82e5a2/src/styles/colors.js#L4

Reply:
Good point. Gonna do better yet and use the `themeColors.border` constant instead.
Located in `src/styles/themes/default.js`